example/gardener-local/gardenlet/values-provider-extensions.yaml

```diff
@@ -0,0 +1,106 @@
+config:
+  gardenClientConnection:
+    kubeconfigSecret:
+      name: gardenlet-kubeconfig
+      namespace: garden
+    bootstrapKubeconfig:
+      name: gardenlet-kubeconfig-bootstrap
+      namespace: garden
+      kubeconfig: |
+        apiVersion: v1
+        kind: Config
+        current-context: gardenlet-bootstrap
+        clusters:
+        - cluster:
+            insecure-skip-tls-verify: true
+            server: https://gardener-apiserver.relay.svc.cluster.local
+          name: default
+        contexts:
+        - context:
+            cluster: default
+            user: gardenlet-bootstrap
+          name: gardenlet-bootstrap
+        users:
+        - name: gardenlet-bootstrap
+          user:
+            token: 07401d.f395accd246ae52d
+  controllers:
+    shoot:
+      reconcileInMaintenanceOnly: true
+  featureGates:
+    HVPA: true
+    HVPAForShootedSeed: true
+    ManagedIstio: true
+    APIServerSNI: true
+    ReversedVPN: true
+    CopyEtcdBackupsDuringControlPlaneMigration: true
+    DefaultSeccompProfile: true
+    CoreDNSQueryRewriting: true
+  logging:
+    enabled: true
+    loki:
+      enabled: true
+    shootNodeLogging:
+      shootPurposes:
+      - "infrastructure"
+      - "production"
+      - "development"
+      - "evaluation"
+  seedConfig:
+    metadata:
+      # Set via skaffold.yaml
+      name: ""
+    spec:
+      # backup:
+      #   provider: gcp
+      #   secretRef:
+      #     name: backup-gcp
+      #     namespace: garden
+      dns:
+        provider:
+          secretRef:
+            # Set via skaffold.yaml
+            name: ""
+            namespace: garden
+          # Set via skaffold.yaml
+          type: ""
+      ingress:
+        controller:
+          kind: nginx
+        # Set via skaffold.yaml
+        domain: ""
+      networks:
+        blockCIDRs:
+        - 169.254.169.254/32
+        # Set via skaffold.yaml
+        pods: ""
+        # Set via skaffold.yaml
+        nodes: $NODES_CIDR
+        services: ""
+        shootDefaults:
+          # CIDR should not overlap with default shoot CIDRs of Gardener landscapes
+          # pods: 100.64.0.0/12
+          # services: 100.104.0.0/13
```

Comment:
I don't understand this comment (of course, I do, but newbies are getting confused). Remove these 3 lines.